components/Tabs2/tests/unit/MDCTabBarViewTests.m

```diff
@@ -17,12 +17,131 @@
 #import "MDCTabBarView.h"
 
 @interface MDCTabBarViewTests : XCTestCase
+
+@property(nonatomic, strong) MDCTabBarView *tabBarView;
+
+@property(nonatomic, strong) UITabBarItem *itemA;
+
+@property(nonatomic, strong) UITabBarItem *itemB;
+
+@property(nonatomic, strong) UITabBarItem *itemC;
+
 @end
 
 @implementation MDCTabBarViewTests
 
+- (void)setUp {
+  [super setUp];
+
+  _tabBarView = [[MDCTabBarView alloc] init];
+  _itemA = [[UITabBarItem alloc] initWithTitle:@"A" image:nil tag:0];
+  _itemB = [[UITabBarItem alloc] initWithTitle:@"B" image:nil tag:0];
+  _itemC = [[UITabBarItem alloc] initWithTitle:@"C" image:nil tag:0];
+}
+
+- (void)tearDown {
+  _itemA = nil;
+  _itemB = nil;
+  _itemC = nil;
+  _tabBarView = nil;
+
+  [super tearDown];
+}
+
 - (void)testInitCreatesObject {
-  XCTAssertNotNil([[MDCTabBarView alloc] init]);
+  // Given
```

Comment:
Tip: I believe this would be the "When" of the test.  You're testing what the result of creating an object is.

Reply:
Sure!